### PR TITLE
Add timthelion's opinion on the dweb:/ipfs vs `/ipfs/` debate

### PR DESCRIPTION
**Pull request by timthelion**

### CODE REVIEW

File: addresses-scheme/readme.md [hunk 1]
```diff
@@ -70,3 +70,192 @@ In [this cryptic comment](https://github.com/ipfs/in-web-browsers/issues/28#issu
...
+proc/ root/ run/ sbin/ sctp/ srv/ sys/ tmp/ udt/ unix/ usr/ utp/ var/ vmlinuz@
+````
+[multiaddr-protocols](https://github.com/multiformats/multiaddr/blob/master/protocols.csv)
+"""
+
+While this may be a misunderstanding on his part, other users may be confused as well.
+
+@timthelion suggests, that if the ipfs devs insist on mixing special paths into the POSIX `/` namespace, they at least use a subdirectory and not polute all of the filesystem root `/`.
+
+### @timthelion's compromise: place all `multiaddr` filesystems under a special directory `/webfs/` and treat the prefix `/webfs/` as a magic path prefix
```
Comment (lidel) [inline addresses-scheme/readme.md]: I kinda like this aesthetically. No matter what prefix is picked, it looks better than anything with `://` 
`/webfs/ipfs/QmT272yei1Zn1eAUq5P9nZyeaKP4oJmVv7CbYvUPyk3aLj/hobby.jpg`
`/dweb/ipfs/QmT272yei1Zn1eAUq5P9nZyeaKP4oJmVv7CbYvUPyk3aLj/hobby.jpg`
`/x/ipfs/QmT272yei1Zn1eAUq5P9nZyeaKP4oJmVv7CbYvUPyk3aLj/hobby.jpg`

**Comment by flyingzumwalt** [comment]
This design discussion has been going on for a long time, with many lengthy discussions in github issues, such as the discussion you brought up in https://github.com/ipfs/ipfs/issues/227 (and now expanded on in this PR).

My goal with this document is to condense all of the discussions on the topic, with little synopses of the important points and links out to the longer design discussions of those points, so that everyone can be on the same page and so that there is a stable reference point for getting up to speed.  Think of it as a reference resource for navigating the discussions rather than the venue where the detailed design discussions occur. The detailed design discussions should continue to happen in GH issues like yours and in PRs with draft specs, like [this PR](https://github.com/ipfs/specs/pull/139). 

I've added a "Criticisms of this Approach"  and "Strengths of this Approach" sub-section to each of the sections in the document and also added a "Proposed Compromises" section at the bottom. Based on that, can you suggest a concise way to describe your criticisms -- a bullet point/heading and a paragraph synopsis of each of your points, with links pointing to the full, detailed discussion? Please find a descriptive heading for your criticism. "@timthelion's viewpoint" does not help people know why you're posing the criticism.

BTW - you will need to rebase your branch (or fork again). When I added the new headings it made a merge conflict with your branch.

**Comment by flyingzumwalt** [comment]
I do want your comments in this document, but I need you to make the comments more compact. The document is currently 135 lines long. That's a condensation of dozens of comments on github issues and PRs. I want to include your points, with links to the longer discussions, but we need to do that in a couple paragraphs, not 190 lines.

**Comment by flyingzumwalt** [comment]
Are you on IRC? How can I help you make sure your comments and examples are included in this discussion? I want to keep this specific document short, but I want these examples and questions to be posed so they can be addressed.

**Pull request closed without merging**